zio/zngio/trailer.go

```diff
@@ -125,7 +125,6 @@ func findCandidate(b []byte, off int) int {
 		}
 	}
 }
-
```

Comment:
Keep me!

Reply:
...I could have sworn I already removed that line from the diff. 

bd2645d20173936ce509a56eda8c25b4f9b32921